src/it/scala/com.rbmhtechnology.calliope/TransactionalEventProducerSpec.scala

```diff
@@ -359,5 +356,54 @@ class TransactionalEventProducerSpec extends KafkaSpec with MustMatchers with Ty
         flowProbe.expectMsg(Event("1", "agg1"))
       }
     }
+    "when stopped" must {
+      val stopSettings = settings.withReadInterval(100.millis).withDeleteInterval(2.seconds)
+
+      "not produce new events" in {
+        val (topic, consumer) = topicConsumer()
+        val eventStore = WritableEventStore()
+        val writer = runTransactionalEventProducer(topic, eventStore, stopSettings)
+
+        whenReady(producer.stop(), timeout(Span(1, Seconds))) { _ =>
```

Comment:
How about using the implicit `PatienceConfig` parameter here and below instead of repeatedly specifying the timeout?